src/dlgConnectionProfiles.cpp

```diff
@@ -817,6 +821,9 @@ void dlgConnectionProfiles::slot_item_clicked(QListWidgetItem* pItem)
         if (profile_name == QStringLiteral("Reinos de Leyenda")) {
             host_url = QStringLiteral("reinosdeleyenda.es");
         }
+        if (profile_name == QStringLiteral("Test_Profile")) {
+            host_url = QStringLiteral("localhost");
```

Comment:
We won't be able to change this for existing profiles once made - so let's go with `mudlet.org` and we'll stand something up later.

Reply:
Makes sense. Should I change the port or will 23 still be good?

Reply:
23 is good.